sniffles/petabi_rule_writer/petabi_rule_writer.py

```diff
@@ -0,0 +1,336 @@
+import getopt
+import sys
+from datetime import date
+import re
+import random
+import codecs
+import sniffles.pcrecomp
+from sniffles.nfa import PCRE_OPT
+
+"""
+   petabi_rule_writer. This is a simple petabi rule writer.
+   It takes a regex file and simple parameters to create petabi rule
```

Comment:
When there is a usage it sometimes pays to just add a little introduction to the usage and let that serve as documentation.  This way, the documentation is in one place and is easier to maintain.  I know Sniffles doesn't do that but Sniffles is a bit older and dates back to when I still thought it was a good idea to add comments to everything.  Anyway, a short comment is fine, but if the comments pertain to usage, it should probably be in the usage.